evap/staff/importers.py

```diff
@@ -427,13 +490,24 @@ def check_evaluation_data_correctness(self, semester):
         missing_degree_names = set()
         missing_course_type_names = set()
         for evaluation_data in self.evaluations.values():
-            if Course.objects.filter(semester=semester, name_en=evaluation_data.name_en).exists():
+            merge_hindraces = evaluation_data.check_for_existing_course(semester)
+            if merge_hindraces:
                 self.errors[ImporterError.COURSE].append(
-                    _("Course {} does already exist in this semester.").format(evaluation_data.name_en)
+                    format_html(
+                        _(
+                            "Course {} ({}) does already exist in this semester, but the courses can not be merged for the following reasons:<br /> - {}."
+                        ).format(
+                            evaluation_data.name_en,
+                            evaluation_data.name_de,
+                            "<br /> - ".join(merge_hindraces),
+                        )
```

Comment:
To get rid of the `<br />` duplication we could:
```suggestion
                        _(
                            "Course {} ({}) does already exist in this semester, but the courses can not be merged for the following reasons:{}."
                        ).format(
                            evaluation_data.name_en,
                            evaluation_data.name_de,
                            "".join(f"<br /> - {msg}" for msg in merge_hindraces),
                        )
```